colcon_core/verb/build.py

```diff
@@ -89,6 +92,10 @@ def add_arguments(self, *, parser):  # noqa: D102
             '--install-base',
             default='install',
             help='The base path for all install prefixes (default: install)')
+        parser.add_argument(
+            '--start_path',
+            default=DEFAULT_START_PATH,
+            help='The directory where build verb is invoked in')
```

Comment:
Why should this be a command line argument? How does it effect all the other command line argument with relative paths?